.github/workflows/CI-e2e-notebooks.yml

```diff
@@ -115,7 +116,13 @@ jobs:
           name: requirements-dev.txt
           path: raiwidgets/installed-requirements-dev.txt
 
-      - name: Run widget tests
+      - if: ${{ matrix.flights != 'noFlights' }}
+        name: Run widget tests
+        shell: bash -l {0}
+        run: yarn e2e-widget -n ${{ matrix.notebook }} -f ${{ matrix.flights }}
+
+      - if: ${{ matrix.flights == 'noFlights' }}
+        name: Run widget tests
```

Comment:
all of these extra logic seems realtted to hardcode value "noFlights", can we remove this hard coded value?

Reply:
The problem is that I have to add something to the matrix. No flights means the absence of that argument. I may be missing something here but I think the yml syntax for GHA isn't quite expressive enough to simplify it.

Reply:
I can't pass an empty arg to e2e-widget.js because it expects me to pass a flight argument to `-f`. I can pass nothing (including no `-f` argument), of course, which is what I'm doing in this if-case. But for that I have to have the if-case. For that, it doesn't really matter if I call it `''` or `'noFlights`. In fact, the latter is more descriptive. Wdyt?

Reply:
Fixed this to work with empty string as discussed.